WebMVC/Services/EventPicService.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using WebMVC.Infrastructure;
+
+namespace WebMVC.Services
+{
+    public class EventPicService
```

Comment:
Instead of having an another EventPicService class, I felt like we can do below steps to reduce the code.
Code changes in WebMVC CatalogService:
                        private readonly string _defaultPicBaseUri;
                     _defaultPicBaseUri=  $"{config["CatalogUrl"]}/api/pic/"

Add GetDefaultImageUrl(_defaultPicBaseUri) inside the CatalogService and call GetDefaultImage method in ApiPaths from here.
Add GetDefaultImage method insode the ApiPaths class itself, and call it from CatalogService directly instead of routing through antoher PicService class and having another Pic class inside the ApiPaths class.

Just give it a thought.

Reply:
Yes can do as per your suggestion but I followed standard practice and if in future we added the feature of uploading an Image then we can use same service.

Reply:
ok, if you say so.. I am good with PR.. approving it..